core/src/term/mod.rs

```diff
@@ -1234,9 +1234,9 @@ pub enum UnaryOp {
 
     /// Typecast an enum to a larger enum type.
     ///
-    /// `Embed` is used to upcast enums. For example, if a value `x` has enum type `a | b`, then
-    /// `embed c x` will have enum type `a | b | c`. It only affects typechecking as at runtime
-    /// `embed someId` act like the identity.
+    /// `EnumEmbed` is used to upcast enums. For example, if a value `x` has enum type `a | b`,
+    /// then `enum/embed c x` will have enum type `a | b | c`. It only affects typechecking as at
```

Comment:
I think this was written a very long time ago, so let's use the current primop syntax in Nickel snippets.

```suggestion
    /// then `%enum/embed% c x` will have enum type `a | b | c`. It only affects typechecking as at
```